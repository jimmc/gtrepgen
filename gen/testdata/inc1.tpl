File from ref dir 1.
{{include "inc2"}}
