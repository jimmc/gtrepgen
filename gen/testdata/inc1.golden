File from ref dir 1.
File from second ref dir.

